src/Console/Command/DescribeCommand.php

```diff
@@ -209,6 +210,14 @@ private function describeRule(OutputInterface $output, $name)
                     $line .= '<comment>required</comment>';
                 }
 
+                if ($option instanceof DeprecatedFixerOption) {
+                    $line .= '. <error>DEPRECATED</error>: '.preg_replace(
+                        '/(`.+?`)/',
+                        '<info>$1</info>',
+                        lcfirst(OutputFormatter::escape($option->getDeprecationMessage()))
```

Comment:
shouldn't the escape be the last operation here (after the `lcfirst`)?

Reply:
Escaping doesn't have any impact on alphanumeric characters so the result would be the same.

Reply:
it used to be and it might be again in the future, I think it is best not to rely on the implementation details about which character(s) are used for the escaping by SF, but than again you are right here about the result being the same for now so :)

Reply:
You're right, maybe worth creating a dedicated PR as we do the same in several other places.